packages/examples/packages/manage-state/src/utils.ts

```diff
@@ -43,15 +50,23 @@ export async function getState(): Promise<State> {
  * browser.
  *
  * @param newState - The new state of the snap.
+ * @param encrypted - An optional flag to indicate whether to use encrypted storage or not.
  * @see https://docs.metamask.io/snaps/reference/rpc-api/#snap_managestate
  */
-export async function setState(newState: State) {
+export async function setState(newState: State, encrypted?: boolean) {
   await snap.request({
     method: 'snap_manageState',
 
-    // For this particular example, we use the `ManageStateOperation.UpdateState`
-    // enum value, but you can also use the string value `'update'` instead.
-    params: { operation: ManageStateOperation.UpdateState, newState },
+    params: {
+      // For this particular example, we use the `ManageStateOperation.UpdateState`
+      // enum value, but you can also use the string value `'update'` instead.
+      operation: ManageStateOperation.UpdateState,
+      newState,
+      // By default all state is encrypted, but you can choose to not encrypt it
+      // To do this you may set this flag to false
+      // This will use a separate unencrypted storage from the encrypted state
```

Comment:
```suggestion
      // By default all state is encrypted, but you can choose to not encrypt it.
      // To do this you may set this flag to false.
      // This will use a separate unencrypted storage from the encrypted state.
```